src/assets/styles/layout.scss

```diff
@@ -19,16 +19,6 @@
   padding-right: 0 !important;
 }
 
-#breadcrumb > .container {
-  max-width: 1360px; // should match auth-web, etc
-}
-
-#breadcrumb-back-btn {
-  width: 28px;
-  height: 28px !important;
-  background-color: white !important;
-}
```

Comment:
Wait - so this CSS broke the styling? But it fixes it if you put it in App.vue?

Could you explain why CSS properties got broken and by what?

Reply:
The two specific CSS properties that break it are in overrides.scss: [height: 2.5rem !important](https://github.com/bcgov/namerequest/blob/93f82f43983165d8661024fb50ebe4d56ddd9343/src/assets/styles/overrides.scss#L106) and [background-color: $app-blue !important;](https://github.com/bcgov/namerequest/blob/93f82f43983165d8661024fb50ebe4d56ddd9343/src/assets/styles/overrides.scss#L109). The css removed from here was the temporary fix used in [16633](https://github.com/bcgov/namerequest/pull/618) (also see [this discussion](https://github.com/bcgov/namerequest/pull/618/commits/f738f6235b0b4500c9a5a94afe27bae799584d49#r1253348992)). 

I believe the confusion is because when I created this ticket, I was not sure what the problem was but believed it was in the shared component. Since the issue was discovered to be specific to this repo, and the problem was technically already fixed, I think the scope of this ticket has changed to more of a clean-up ticket.

Reply:
On the first point, I don't understand why having this CSS in layout.scss causes the problem, but moving it to App.vue fixes it.

One the second point, yes, I agree with you.

Reply:
Hey Sev, just to clarify the first point, the css removed here is not the cause of the problem. If you look at namerequest ui on the feature branch you'll see the button looks fine even without the changes from this PR. (For example this temp url from Ketaki's PR when the environment get's fixed: https://namerequest-dev--pr-639-c9rqzr4l.web.app/).

This css is the solution, I just didn't like having it in the layout.css.

Reply:
Oh, right! You fixed this previously. Now you're moving the CSS to where it's used. I like it. Resolving.